packages/flutter/lib/src/services/predictive_back_event.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// Enum representing the edge from which a swipe starts in a back gesture.
+///
+/// This is used in [PredictiveBackEvent] to indicate the starting edge of the
+/// swipe gesture.
+enum SwipeEdge {
+  /// Indicates that the swipe gesture starts from the left edge of the screen.
+  left,
+
+  /// Indicates that the swipe gesture starts from the right edge of the screen.
+  right,
+}
+
+/// Object used to report back gesture progress in Android.
+///
+/// Holds information about the touch event, swipe direction and the animation
+/// progress that predictive back animations should seek to.
```

Comment:
For my understanding, what does `seek to` mean in this context?

Reply:
I was thinking of it as in "seeking" a video forward/backward in time by grabbing the little progress indicator. I think it's an accurate term here, but I see your confusion. CC @maRci002.

It's repeated a few times in another paragraph below. I think I'll replace this usage with "follow" and leave the ones below, since they have more context to understand it with?